cmake/packaging.cmake

```diff
@@ -0,0 +1,56 @@
+set(PACKAGE_PROJECT_NAME        "OpenXRay")
+set(PACKAGE_PROJECT_DESCRIPTION "OpenXRay is an improved version of the X-Ray Engine, the game engine used in the world-famous S.T.A.L.K.E.R. game series by GSC Game World.")
+set(PACKAGE_PROJECT_CONTACT     "OpenXRay <openxray@yahoo.com>")
+set(PACKAGE_PROJECT_VERSION     "1.6.02")
+set(PACKAGE_PROJECT_HOME_URL    "https://github.com/OpenXRay/xray-16")
+
+if (UNIX)
+    if (EXISTS "${CMAKE_ROOT}/Modules/CPack.cmake")
+        set(CPACK_PACKAGE_NAME ${PACKAGE_PROJECT_NAME})
+        set(CPACK_FILE_NAME "openxray")
+        set(CPACK_PACKAGE_VERSION ${PACKAGE_PROJECT_VERSION})
+        set(CPACK_PACKAGE_CONTACT ${PACKAGE_PROJECT_CONTACT})
+        set(CPACK_PACKAGE_DESCRIPTION ${PACKAGE_PROJECT_DESCRIPTION})
+        set(CPACK_PACKAGE_DESCRIPTION_SUMMARY ${CPACK_PACKAGE_DESCRIPTION})
+        set(CPACK_STRIP_FILES TRUE)
+        set(CPACK_SOURCE_IGNORE_FILES "/.gitattributes")
+        set(CPACK_RESOURCE_FILE_README ${PROJECT_SOURCE_DIR}/README.md)
+        set(CPACK_RESOURCE_FILE_LICENSE ${PROJECT_SOURCE_DIR}/License.txt)
+
+        # --- SELECT PROPER CPACK GENERATOR ---
+        if (DEBIAN_FOUND)
+            set(CPACK_GENERATOR DEB)
+
+            set(CPACK_DEBIAN_FILE_NAME "DEB-DEFAULT")
+            set(CPACK_DEBIAN_PACKAGE_CONTROL_STRICT_PERMISSION TRUE)
+            set(CPACK_DEBIAN_REVISON "ubuntu-bionic-a1") # TODO: This one need to be detected dynamically
```

Comment:
Not sure about existing and helpfulness of this parameters at all, because official documentation not include them https://cmake.org/cmake/help/v2.8.0/cmake.html#module:CPackDeb don't know from where it was taken. It was added by @eagleivg in this commit 7869f7 you should ask him about that. Maybe it was taken from that article: https://pmateusz.github.io/linux/2017/06/30/linux-secure-apt-repository.html